package.json

```diff
@@ -10,7 +10,7 @@
     "@patternfly/react-styles": "5.3.1",
     "@patternfly/react-table": "5.3.3",
     "@patternfly/react-tokens": "5.3.1",
-    "deep-equal": "2.2.3",
+    "dequal": "2.0.0",
```

Comment:
IMHO dequal/lite is enough. The size difference is marginal, but it also does less checks and thus is faster. We don't need to compare ArrayBuffers and such, this is mostly just about comparing simple dicts.

Not a blocker of course, if this works, let's go!

Reply:
Is that a separate module which I can put in `package.json`? Uh

Reply:
```
[jelle@t14s][~/projects/cockpit/dequal-tryout]%ls node_modules/dequal/
dist/  index.d.ts  license  lite/  package.json  readme.md
```

Is what we have now?

Reply:
```
npm error Invalid package name "dequal/lite" of package "dequal/lite@2.0.3": name can only contain URL-friendly characters.
```
So it seems we already have `lite` :)

Reply:
Yes, don't confuse the npm package name (dequal) with the `import` module names (dequal or dequal/lite). We "have" it in the sense of "transitive build dependency", but we don't have it yet in the runtime bundles in dist/.

Reply:
Right, and I did that comment against the wrong line. Anyway, all good now!